pkg/start/start.go

```diff
@@ -153,16 +178,97 @@ func (o *Options) Run() error {
 	return nil
 }
 
+func (o *Options) handler() http.Handler {
+	mux := http.NewServeMux()
+	mux.Handle("/metrics", promhttp.Handler())
+	return mux
+}
+
+func (o *Options) makeTLSConfig() (*tls.Config, error) {
+	if !fileExists(o.ServingCertFile) {
+		return nil, errors.New(o.ServingCertFile + " does not exist")
+	}
+
+	if !fileExists(o.ServingKeyFile) {
+		return nil, errors.New(o.ServingKeyFile + " does not exist")
+	}
+
+	// Load the initial certificate contents.
+	certBytes, err := ioutil.ReadFile(o.ServingCertFile)
+	if err != nil {
+		return nil, err
+	}
+	keyBytes, err := ioutil.ReadFile(o.ServingKeyFile)
+	if err != nil {
+		return nil, err
+	}
+	certificate, err := tls.X509KeyPair(certBytes, keyBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	return crypto.SecureTLSConfig(&tls.Config{
+		GetCertificate: func(_ *tls.ClientHelloInfo) (*tls.Certificate, error) {
+			return &certificate, nil
+		},
+		ClientAuth: tls.RequestClientCert,
```

Comment:
Do we care about client certs?  [The default `NoClientCert`][1] seems reasonable to me.

[1]: https://golang.org/pkg/crypto/tls/#Config

Reply:
changed in abf62a5 to use default NoClientCert